code/mob/living/intangible/wraith.dm

```diff
@@ -405,12 +405,25 @@
 	equipped()
 		return 0
 
-	click(atom/target)
+	click(atom/target, params)
 		if (src.targeting_ability)
 			return ..()
-		if (!density)
+		if (src.client && src.client.check_key(KEY_POINT))
+			src.point_at(target, text2num(params["icon-x"]), text2num(params["icon-y"]))
+
+		else if (!density)
 			src.examine_verb(target)
 
+		//Stolen from /mob/click because
```

Comment:
why doesn't this just call the parent anyways? seems smelly to reimplement code from the parent when we could just be... calling that

Reply:
IDK, they did something weird with the order of things and I didn't want to figure it out. I suppose I should